frontend/src/scenes/billing/PlanComparison.tsx

```diff
@@ -127,7 +127,7 @@ export const PlanComparison = ({
     const { reportSurveyShown, setSurveyResponse } = useActions(billingProductLogic({ product }))
     const { featureFlags } = useValues(featureFlagLogic)
 
-    const ctaAction = featureFlags[FEATURE_FLAGS.SUBSCRIBE_TO_ALL_PRODUCTS] === 'test' ? 'Upgrade' : 'Subscribe'
+    const ctaAction = billing?.subscription_level === 'custom' ? 'Subscribe' : 'Upgrade'
```

Comment:
We should run a small A/B test to see which language works best. "Add card" vs "Subscribe" vs "Upgrade" now that it's a single sub. I think bianca did this at some point but I don't remember the result.

Reply:
Yep! That's on my list :)